portal-web/test/functional/com/liferay/portalweb/paths/portlet/roles/RolesAssignMembers.path

```diff
@@ -20,7 +20,7 @@
 </tr>
 <tr>
 	<td>ADD_ASSIGNEE_TABLE_NAME</td>
-	<td>//div[contains(@id,'assigneesSearchSearchContainer')]//tr[contains(.,'${key_assigneeName}')]/td[2]</td>
+	<td>//div[contains(@id,'assigneesSearchSearchContainer')]//tr[contains(.,'${key_assigneeName}')]/td[3]</td>
```

Comment:
@alanafergs Hey Alana, this locator is widely used. Did you check if this change will affect other tests?